lib/runcible/base.rb

```diff
@@ -45,13 +45,18 @@ def call(method, path, options = {})
       if clone_config[:oauth]
         headers = add_oauth_header(method, path, headers) if clone_config[:oauth]
         headers['pulp-user'] = clone_config[:user]
-        client = RestClient::Resource.new(clone_config[:url], client_options)
+      elsif clone_config[:cert_auth]
+        fail Runcible::ConfigurationUndefinedError, "Missing necessary ssl information." unless (clone_config[:ssl_client_cert] && clone_config[:ssl_client_key] && clone_config[:ssl_ca_file])
+        client_options[:ssl_client_cert] => clone_config[:ssl_client_cert]
+        client_options[:ssl_client_key] => clone_config[:ssl_client_key]
+        client_options[:ssl_ca_file] => clone_config[:ssl_ca_file]
```

Comment:
This might slightly conflict naming wise with https://github.com/Katello/runcible/blob/master/lib/runcible/base.rb#L167 ? As in, I believe they represent the same thing and we should reconcile the two.